composables/useScenarioOptions.ts

```diff
@@ -1,27 +1,61 @@
-    const axis = toValue(parameterAxis);
-    if (!appStore.currentScenario.parameters || !axis) {
+    if (!appStore.currentScenario.parameters || !axis.value) {
       return undefined;
-    } else if (axis.parameterType === TypeOfParameter.Numeric) {
-      const baselineValue = appStore.currentScenario.parameters[axis.id];
+    } else if (axis.value.parameterType === TypeOfParameter.Numeric) {
+      const baselineValue = appStore.currentScenario.parameters[axis.value.id];
       // TODO: (jidea-229) description should say whether the value is a default, min, max; or empty if user-provided.
-      // TODO: (jidea-229) For numeric options, do (locale-based) comma-separation of thousands.
-      return { id: baselineValue, label: baselineValue, description: "" } as ParameterOption;
-    } else {
-      return axis.options?.find((o) => {
-        return o.id === appStore.currentScenario.parameters![axis.id];
-      });
+      return { id: baselineValue, label: humanReadableInteger(baselineValue), description: "" } as ParameterOption;
+    } else if (axis.value.id && appStore.currentScenario.parameters) {
+      return axis.value.options?.find(o => o.id === appStore.currentScenario.parameters![axis.value!.id]);
+    }
+  });
+
+  const dependedOnParamOptionLabel = computed(() => {
+    const dependedOnParamId = axis.value?.updateNumericFrom?.parameterId;
+    const dependedOnParamValue = dependedOnParamId ? appStore.currentScenario.parameters?.[dependedOnParamId] : undefined;
+    if (dependedOnParamId && dependedOnParamValue && appStore.metadata?.parameters) {
+      return appStore.metadata?.parameters
+        .find(p => p.id === dependedOnParamId)
+        ?.options
+        ?.find(o => o.id === dependedOnParamValue)
+        ?.label;
+    }
+  });
+
+  const predefinedNumericOptions = computed(() => {
+    const dependentRange = getRangeForDependentParam(axis.value, appStore.currentScenario.parameters);
+
+    if (dependentRange) {
+      const min = dependentRange.min.toString();
+      const defaultVal = dependentRange.default.toString();
+      const max = dependentRange.max.toString();
+
+      return [
+        { id: min, label: humanReadableInteger(min), description: `Minimum for ${dependedOnParamOptionLabel.value}` },
+        { id: defaultVal, label: humanReadableInteger(defaultVal), description: `Default for ${dependedOnParamOptionLabel.value}` },
+        { id: max, label: humanReadableInteger(max), description: `Maximum for ${dependedOnParamOptionLabel.value}` },
+      ] as ParameterOption[];
     }
   });
 
   const nonBaselineOptions = computed(() => {
-    return toValue(parameterAxis)?.options?.filter(({ id }) => {
+    if (!axis.value) {
+      return [];
+    }
+    if (axis.value?.parameterType === TypeOfParameter.Numeric) {
+      return predefinedNumericOptions.value?.filter(o => o.id !== baselineOption.value?.id) || [];
+      // TODO - exclude any option that is equal to the baseline?
```

Comment:
Yeah, I guess so, and maybe give some feedback to the user about why they can't manually add the baseline value.

Reply:
Again I should have deleted the comment since the preceding line achieves what it says, and the ScenarioSelect says "...is already selected" if you try it.